### ISSUE TITLE
Tic-Tac-Toe(Admin Choice - Level X) 

### ISSUE THREAD

**Issue by SarthakKeshari** [SarthakKeshari/Java-Questions-and-Solutions#171]
### Enter your question - 
Write a program for end user so that he/she can play tic-tac-toe game with computer or game can be played by two end users.

Fulfilling all the below conditions is necessary for PR to be accepted - 
Game UI and working -
### Enter link to the question(if question belongs to any online platform) - 
No link

### Tags for the question(eg - Array, Basic, Stack, etc.) - 
Application Based

**Comment by SarthakKeshari** [comment]
Everyone,
Deadline - 08/10/2021

**Comment by mtwn105** [comment]
Can I do it?

**Comment by SarthakKeshari** [comment]
@mtwn105,
Sure go ahead.

**Comment by mtwn105** [comment]
Hi @SarthakKeshari opened PR for this :D

**Comment by mtwn105** [comment]
Sure. I had a question that this repo is considered for "Hacktoberfest" or not because many other DSA repos were excluded

**Comment by SarthakKeshari** [comment]
@mtwn105,
Yes, I got the info regarding that. Since the name only says "Java-Questions-and-Solutions" but plan is to work with website building. Creating a knowledge hub of Java. Initial part was the DSA one so that people get to begin with open source. This repo aims to be beginner friendly.
